engine/lib/phx/src/engine/mod.rs

```diff
@@ -305,8 +301,12 @@ impl Engine {
         let entry_point_path = PathBuf::from(entry_point);
 
         if !entry_point_path.exists() {
-            // TODO: do we really need this magic?
-            std::env::set_current_dir("../").expect("Cannot change folder to parent");
+            // If we can't find it, set the current dir to one above the executable path and try that instead.
+            let mut dir = std::env::current_exe().expect("Cannot get the path to the executable");
```

Comment:
I added this so that we always try relative to the actual executable, rather than relative to whatever the working directory happens to be right now.